src/Components/HousingFinderList.jsx

```diff
@@ -13,16 +13,16 @@ import { ListItemLink } from "../ui-kit/ListItemLink";
 const HousingFinderList = ({ resources }) => {
   const classes = useResourcesStyles();
   return (
-    <List dense>
+    <List dense style={{ borderStyle: "solid", width: "100%" }}>
```

Comment:
Is there a reason why the `Organizations for help finding housing` content has a black border and the others don't?

Reply:
It was there to check the boundaries and should be removed.